src/backend/marsha/settings.py

```diff
@@ -275,6 +273,15 @@ class Base(Configuration):
     XMPP_JWT_AUDIENCE = values.Value("marsha")
     XMPP_DOMAIN = values.Value(None)
 
+    # LIVE SETTINGS
+    NB_DAYS_BEFORE_DELETING_LIVE_RECORDINGS = values.Value(14)
+    NB_DAYS_KEEPING_LIVE_IDLE = values.Value(7)
+    LIVE_PLAYLIST_WINDOW_SECONDS = values.Value(10)
+    LIVE_SEGMENT_DURATION_SECONDS = values.Value(4)
+    LIVE_FRAMERATE_NUMERATOR = values.Value(24000)
+    LIVE_FRAMERATE_DENOMINATOR = values.Value(1000)
+    LIVE_GOP_SIZE = values.Value(4)
```

Comment:
Maybe we should explain those variables in UPGRADE.md?

Reply:
I upgraded `docs/env.md`

Reply:
Nice :)